src/pages/ClaimedPage.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import SingleBook from "../components/SingleBook";
+import { Link } from "react-router-dom";
+
+export default function Claimed() {
```

Comment:
This component is almost exactly the same as Home. Do you think there is anyway you could combine them into a single component that can either show claimed or unclaimed?